css/components/question-card.css

```diff
@@ -0,0 +1,44 @@
+.question-card {
+  margin: 100px 80px 100px 80px;
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.question-card__answer-container {
+  margin: 20px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.question-card__button {
+  margin: 20px;
+  padding: 5px;
+  color: var(--primary-color);
+  background-color: #fff;
+  border-radius: 6px;
+  border: 1px solid var(--primary-color);
+}
+
+.question-card__text {
+  margin: 10px;
+}
+
+.question-card__list {
+  margin: 10px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.question-card__list-item {
+  margin: 0px 10px 10px 10px;
+  content: 10px;
```

Comment:
for what do you use content: 10px?

you usually use content for ::before or ::after elements

https://developer.mozilla.org/en-US/docs/Web/CSS/content?retiredLocale=de